box2d/src/test/kotlin/ktx/box2d/worldsTest.kt

```diff
@@ -218,4 +218,39 @@ class WorldsTest : Box2DTest() {
     assertEquals(0f, RayCast.TERMINATE, tolerance)
     assertEquals(-1f, RayCast.IGNORE, tolerance)
   }
+
+  @Test
+  fun `should query for overlapping fixtures with AABB and stop`() {
+    val world = createWorld()
+    val matchingEdge1 = world.body {}.edge(from = Vector2.Zero, to = Vector2(0f, 2f)) {}
+    val matchingEdge2 = world.body {}.edge(from = Vector2(1f, 0f), to = Vector2(1f, 2f)) {}
```

Comment:
Is it still necessary to use empty `{}` instead of just `()`? I'm pretty sure KTX Box2D provides default init blocks now - if this is impossible, this is a regression or omission in the API.

Reply:
Looks like they defaults are missing for `World.body`

Reply:
Good catch, I'll add it today.